cpmonitor/views.py

```diff
@@ -262,20 +272,41 @@ def administration_checklist_view(request, city_slug):
     return render(request, "administration_checklist.html", context)
 
 
-def _get_administration_checklist(city) -> dict:
```

Comment:
You could try to use one function for both checklists. Also, I would prefer not to use indices in the templates. I have not tested this and also not checked if that helps in the template, but what do you think of this:
```python
def _convert_checklist(checklist):
    try:
        checklist_fields = checklist._meta.get_fields()
    except Model.DoesNotExist:
        return []

    checklist_items = [
        field
        for field in checklist_fields
        if field.attname not in ["city_id", "id"] and "_radionale" not in field.attname
    ]
    return {
        item.verbose_name: {
            "is_checked": getattr(checklist, item.attname),
            "rationale": getattr(checklist, item.attname + "_rationale"),
        }
        for item in checklist_items
    }
```

Reply:
That requires the schema `<checklist_item>_rationale` for all entries.

Reply:
I like your type annotations, though. Was just too lazy to put them in myself.

Reply:
I like the dictionary way and adapted the code